cmd/remote/folder-package.go

```diff
@@ -0,0 +1,43 @@
+package remote
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/criteo/command-launcher/internal/command"
+)
+
+type folderPackage struct {
+	defaultPackage
+	sourceDir string
+}
+
+func CreateFolderPackage(folder string) (command.Package, error) {
```

Comment:
I don't think we need a folder package. The "remote" module abstracts the command launcher package on a remote server. There could be packages in form of git repo or zip file, but I am not sure the folder package.